libpod/volume.go

```diff
@@ -294,3 +298,55 @@ func (v *Volume) Unmount() error {
 func (v *Volume) NeedsMount() bool {
 	return v.needsMount()
 }
+
+// Export volume to tar.
+// Returns a ReadCloser which points to a tar of all the volume's contents.
+func (v *Volume) Export() (io.ReadCloser, error) {
+	v.lock.Lock()
+	err := v.mount()
+	mountPoint := v.mountPoint()
+	v.lock.Unlock()
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		v.lock.Lock()
+		defer v.lock.Unlock()
+
+		if err := v.unmount(false); err != nil {
+			logrus.Errorf("Error unmounting volume %s: %v", v.Name(), err)
+		}
+	}()
```

Comment:
do we actually want to unlock, I realize this is what the code did before but like in what world is it sane to allow volume rm while export/import is running

Reply:
We unlock for commands like `podman cp` so I see no reason not to here?